### PR TITLE
Fix matcher.js final or fallthru.

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**

### CODE REVIEW

File: src/urls.test.js [hunk 1]
```diff
@@ -57,4 +57,20 @@ e/Examples/Wall%20standard%20case/File.ifc')
         },
       })
   })
+
+
+  it('parses localhost file ref', () => {
+    const url = new URL('https://localhost.8090/models/bld/mix.bld')
```
Comment (pablo-mayrgundter) [inline src/urls.test.js:63]: Change . to :

File: src/urls.test.js [hunk 2]
```diff
@@ -57,4 +57,20 @@ e/Examples/Wall%20standard%20case/File.ifc')
...
+    const parsed = parseUrl(url)
+    expect(parsed).toStrictEqual({
+        original: url,
+        type: 'url',
```
Comment (pablo-mayrgundter) [inline src/urls.test.js:67]: change 'url' to enum

File: src/matcher.test.js [hunk 3]
```diff
@@ -48,4 +48,17 @@ describe('matcher', () => {
     expect(cb).toHaveBeenCalledTimes(1)
     expect(fail).not.toHaveBeenCalled()
   })
+
+
+  it('falls thru to final or', () => {
+    const cb = jest.fn()
+    const fail = jest.fn()
+    matcher(
+      'https://localhost8090/models/bld/mix.bld',
```
Comment (pablo-mayrgundter) [inline src/matcher.test.js:57]: :8090

**Review by oo-bldrs — APPROVED** [review]

**Comment by pablo-mayrgundter** [comment]
Per convo, there were some broken unit tests here.  Ended up doing a larger change in #29.

**Pull request closed without merging**